modules/tinymce/changelog.txt

```diff
@@ -5,6 +5,7 @@ Version 5.5.0 (TBD)
     Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011
     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
     Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and its toggle state to be checked #TINY-6032
+    Added the ability to use `colgroup` elements in tables and added a new setting `table_use_colgroups` for toggling the behavior #TINY-6050
```

Comment:
```suggestion
    Added the ability to use `colgroup` elements in tables #TINY-6050
    Added a new setting `table_use_colgroups` for toggling the behavior #TINY-6050
```

Reply:
Hm. @tylerkelly13 is the second line clear enough as a standalone entry?

Reply:
I'd agree, it needs to say something like `toggling whether colgroups are inserted in new tables` or something like that :shrug:

Reply:
@metricjs @lnewson Fixed suggestion?

Reply:
LGTM :+1: We'll just need to get that included now.